cylc/uiserver/handlers.py

```diff
@@ -44,6 +45,7 @@
 
 
 ME = getpass.getuser()
+RE_SLASH = x = re.compile(r'\/+')
```

Comment:
Does the `x` have a purpose?

```suggestion
RE_SLASH = re.compile(r'\/+')
```

Reply:
Sorry, no idea how that got in there.